### DISCUSSION TITLE
How to make role?

### DISCUSSION THREAD

**Discussion by the-hidden-archive** [space-wizards/space-station-14#15628] [Questions]
How to add your role so that it can be selected in the character editor and the late entry window?

**Comment by deltanedas** [comment]
a role or a job? theyre not necessarily the same thing

**Comment by Hubabuber** [comment]
I know for sure that the creation of a role can be associated with folders:
space-station-14\Resources\Prototypes\Roles
here is only about roles, it is advisable to look at them completely.

space-station-14\Resources\Prototypes\Entities\Markers\Spawners
and here are the spawns themselves, which are used when creating the map

space-station-14\Resources\Textures\Markers\jobs.rsi
textures are here 

but for some reason errors come out

**Reply by Hubabuber** [reply]
it would be nice if people would supplement my unfinished instructions
